### PR TITLE
Updated stackdriver exporter

### PR DESCRIPTION
**Pull request by james-bebbington**
**Description:**

- Update stackdriver exporter to v0.13.0
- Add config to allow trace bundle settings to be configured

**Breaking Changes:**

- Nested config `metric_prefix` & `skip_create_metric_descriptor` under `metric` to tidy up the config (happy to revert if people think we should avoid this, but I thought it useful to streamline the config a bit for clarity)
- Trace status codes no longer reflect gRPC as per spec changes: https://github.com/open-telemetry/opentelemetry-specification/pull/1067

cc @nilebox @dashpole

### CODE REVIEW

**Comment by james-bebbington** [comment]
FYI @igorpeshansky @qingling128 - this would have been a good issue to get you to look into in hindsight

**Review by nilebox — APPROVED** [review]

**Review by tigrannajaryan — COMMENTED** [review]
Since the config is change it would be useful to describe it in the CHANGELOG's Breaking Changes section.

Also, see if you can increase the coverage, the new code seems mostly uncovered.

**Comment by james-bebbington** [comment]
> Since the config is change it would be useful to describe it in the CHANGELOG's Breaking Changes section.

Added

> Also, see if you can increase the coverage, the new code seems mostly uncovered.

Extended the tests to cover various bundling configuration options

File: exporter/stackdriverexporter/config.go [hunk 1]
```diff
@@ -39,6 +38,23 @@ type Config struct {
...
+	NumOfWorkers int          `mapstructure:"number_of_workers"`
+}
+
+type TraceConfig struct {
+	BundleDelayThreshold time.Duration `mapstructure:"bundle_delay_threshold"`
+	BundleCountThreshold int           `mapstructure:"bundle_count_threshold"`
+	BundleByteThreshold  int           `mapstructure:"bundle_byte_threshold"`
+	BundleByteLimit      int           `mapstructure:"bundle_byte_limit"`
+	BufferMaxBytes       int           `mapstructure:"buffer_max_bytes"`
+}
```
Comment (bogdandrutu) [inline exporter/stackdriverexporter/config.go:53]: @james-bebbington not for this PR, but bundle options are overkill. We already provide queuing and batching in the collector, and ideally the exporter should not do more than that.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**